src/test/java/SearchTest.java

```diff
@@ -1,9 +1,7 @@
-package test;
```

Comment:
Qui dice che è fuori dal package test, creare un package search in test in cui mettere questa classe